conf/application.conf

```diff
@@ -25,25 +25,24 @@ play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
 
 # Provides an implementation of AuditConnector. Use `uk.gov.hmrc.play.bootstrap.AuditModule` or create your own.
 # An audit connector must be provided.
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuditModule"
+play.modules.enabled += "uk.gov.hmrc.play.audit.AuditModule"
 
-play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoHmrcModule"
+play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
 
 # Provides an implementation of MetricsFilter. Use `uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule` or create your own.
 # A metric filter must be provided
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
 
 # Provides an implementation and configures all filters required by a Platform frontend microservice.
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.FrontendModule"
-play.http.filters = "uk.gov.hmrc.helloworldupscan.config.HelloWorldFilters"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.frontend.FrontendModule"
 
 # Default http client
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 
 # Custom error handler
 play.http.errorHandler = "uk.gov.hmrc.helloworldupscan.config.ErrorHandler"
 
-play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data:"
+play.filters.csp.CSPFilter = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data:"
```

Comment:
The config for csp should be
```
play.filters.enabled += play.filters.csp.CSPFilter
play.filters.csp.directives.default-src = "'self' ..."
```
but it actually looks like CSP was being configured but not enabled?

If we're not using `HelloWorldFilters` - it can be deleted?
It looks like the custom `FilteringCSRFFilter` can also be deleted (bootstrap already enables play's CSRF filter for frontends), but we'd have to replace `#NOCSRF` in app.routes with `+ nocsrf` (https://www.playframework.com/documentation/2.8.x/JavaCsrf#Applying-a-global-CSRF-filter)